config-templates/aws/aws-efs-csi-driver/1.2.2/storage-class-parameters.json

```diff
@@ -0,0 +1,49 @@
+[
+  {
+	"description": "The name of the storage class.",
+	"displayname": "Storage Class Name",
+	"name": "name",
+	"required": "true",
+	"category": "config"
+  }, {
+	"description": "Specify 'true' or 'false' to make the created storage class the default class.",
+	"displayname": "Default",
+	"name": "is-default-class",
+	"default": "false",
+	"required": "false",
+	"category": "config",
+	"type": "text"
+  }, {
+        "description": "Specify the EFS filesystem ID",
+	"displayname": "EFS Filesystem ID",
+	"name": "fsid",
+	"required": "true",
+	"category": "config"
+
+  },{
+        "description": "Specify direcdtory permissions",
+	"displayname": "Directory permission",
+	"name": "directory-perms",
+	"required": "false",
+	"category": "config"
+
+  }, {
+        "description": "Specify the GID range start",
+	"displayname": "GID range start",
+	"name": "gid-range-start",
+	"required": "false",
+	"category": "config"
+  }, {
+        "description": "Specify the GID range end",
+	"displayname": "GID range end",
+	"name": "gid-range-end",
+	"required": "false",
+	"category": "config"
+  }, {
+        "description": "Specify the BasePath",
+	"displayname": "Basepath",
+	"name": "base-path",
+	"required": "false",
+	"category": "config"
+  }
+]
```

Comment:
I think all parameters have some default values. 
https://github.com/kubernetes-sigs/aws-efs-csi-driver/tree/master/examples/kubernetes/dynamic_provisioning
Also see if we can improve description msg by giving more detail for each parameter